### PR TITLE
feat(tableloading): added table of loading cards

### PR DESCRIPTION
**Pull request by FelixBlaisThon**
### Proposed Changes

added a  isCard prop in table loading so we can render loading cards in the same logic as loading table. One addition, is CardSubRow, which is the number of rows inside the card.

### Acceptance Criteria

-   [x] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Comment by FelixBlaisThon** [comment]
Forgot UTs..

File: packages/react-vapor/src/components/loading/components/TableLoading.tsx [hunk 1]
```diff
@@ -2,36 +2,96 @@ import * as classNames from 'classnames';
...
+}: {
+    isCard?: boolean;
+    numberOfColumns?: number;
+    numberOfRow?: number;
+    numberOfSubRow?: number;
+}) => {
+    return (
+        <tbody>
+            {isCard
+                ? _.times(numberOfRow, (nColumn: number) => (
```
Comment (dmgauthier) [inline packages/react-vapor/src/components/loading/components/TableLoading.tsx]: is a ternary the best thing to use here :thinking:  ?

Comment (FelixBlaisThon) [reply packages/react-vapor/src/components/loading/components/TableLoading.tsx]: isCard && would work, but did you have something else in mind?

Comment (rosalie-liu) [reply packages/react-vapor/src/components/loading/components/TableLoading.tsx]: I'd probably extract the two components, then in <tbody> for instance:
`{_.times(numberOfRow, (nColumn: number) => { isCard ? (<CardLoading />) : <RowLoading /> })}`

**Pull request merged**